lib/adapter/freshheads/CssAdapter.ts

```diff
@@ -105,20 +106,24 @@ export default class CssAdapter implements Adapter {
         );
     }
 
-    private validateAllRequiredModulesAreInstalled() {
-        const requiredModules = [
-            'mini-css-extract-plugin',
-            'autoprefixer',
-            'css-loader',
-            'postcss-loader',
-        ];
-
-        requiredModules.forEach(module => {
-            if (!checkIfModuleIsInstalled(module)) {
-                throw new Error(
-                    `The '${module}' needs to be installed for this adapter to work`
-                );
-            }
+    private validateAllRequiredModulesAreInstalled(isProduction: boolean) {
+        const requiredModules: { [module: string]: string } = {
+            'mini-css-extract-plugin': '0.8.0',
+            autoprefixer: '9.7.0',
```

Comment:
je mist de quotes om autoprefixer

Reply:
zie nu wel dat je het op meerdere plekken doet dus waarschijnlijk heb je er een goede reden voor :)

Reply:
Doet `prettier` vanzelf. Kan het ook uitzetten, maar in zekere zin zijn de andere keys de uitzonderingen en hebben ze enkele quotes er omheen omdat ze anders niet als key in een object gebruikt kunnen worden.